subprojects/docs/src/samples/signing/conditional/kotlin/build.gradle.kts

```diff
@@ -16,9 +11,9 @@ version = "1.0-SNAPSHOT"
 extra["isReleaseVersion"] = !version.toString().endsWith("SNAPSHOT")
 
 signing {
-    setRequired(Callable {
+    setRequired {
```

Comment:
:x: I'd keep the parentheses for uniformity with the (common) call sites that still require it. WDYT?